src/events_test.go

```diff
@@ -90,29 +90,33 @@ func Test_alivenessTest_SkipCollect(t *testing.T) {
 
 func Test_healthcheckTest_Pass(t *testing.T) {
 	i := testutils.GetTestingIntegration(t)
+	running := true
 
-	nodeTests := []*data.NodeTest{
-		{
-			Node: &data.NodeData{Name: "node1"},
-			Test: &data.TestData{
-				Status: "ok",
-			},
-		},
+	nodeTests := []*data.NodeData{
+		{Name: "node1"},
+		{Running: &running},
+	}
+	healthcheckTest(i, nodeTests, "testClusterName")
+	assert.Empty(t, i.Entities)
+}
+
+func Test_healthcheckTest_NoRunningField(t *testing.T) {
```

Comment:
I guess the `running` field is supposed to be always there. However, assuming the node ir running when the running field is not there, puzzles me a bit. Why is it the intended behavior?

Reply:
I am just testing it for the reason that, it's a *bool pointer, so it can be empty, for this reason, I added a check of `!= nil ` and I just wanted to test it here. I understand that it will always be there, but ,since it's a pointer, I want to test this behavior.

Reply:
I think I didn't explain myself correctly. I can see why this test is defined, but I don't fully understand why it is assumed that the node is still running even if the `running` field is not there... Shouldn't it be the other way around?

Reply:
I've modified the behavior in the last modification, now if the running field is not present, an event of unknown running status is produced.